lib/message-handler.js

```diff
@@ -80,6 +80,10 @@ export default async function messageHandler(recipeMap, req, res) {
         return 500;
       }
     }
+    if (error.message.startsWith("6 ALREADY_EXISTS")) {
```

Comment:
Du borde kunna använda [`error.code`](https://firebase.google.com/docs/reference/node/firebase.firestore#firestoreerrorcode) här:
```suggestion
    if (error.code.toLowerCase() === "already-exists") {
```